cassandra/cqlengine/connection.py

```diff
@@ -43,7 +43,9 @@
 
 def format_log_context(msg, connection=None, keyspace=None):
     """Format log message to add keyspace and connection context"""
-    connection_info = connection if connection else DEFAULT_CONNECTION
+    connection_info = connection
+    if not connection_info:
```

Comment:
well, I need a string representation of the default connection, for the log... since I changed the DEFAULT_CONNECTION key to an object